sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -327,8 +326,8 @@ abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
     case None => ""
     case Some(v) if v.isEmpty => ""
     case Some(v) =>
-      " The data type of following expressions will be converted in GPU runtime:\n" +
-          v mkString "; "
+      "The data type of following expressions will be converted in GPU runtime: " +
+          (v mkString "; ")
```

Comment:
I know we don't have coding standards for things like this, but I think it would be better to have `v.mkString("; ")` here rather than `(v mkString "; ")` as recommended here - https://docs.scala-lang.org/style/method-invocation.html

Reply:
should be fixed.